airflow/sensors/python.py

```diff
@@ -71,4 +71,4 @@ def poke(self, context: Context) -> PokeReturnValue | bool:
 
         self.log.info("Poking callable: %s", str(self.python_callable))
         return_value = self.python_callable(*self.op_args, **self.op_kwargs)
-        return PokeReturnValue(bool(return_value))
+        return return_value if isinstance(return_value, PokeReturnValue) else PokeReturnValue(bool(return_value))
```

Comment:
Need add [tests](https://github.com/apache/airflow/blob/main/tests/sensors/test_python.py) for this scenario for avoid regression

Reply:
Added a test to check that the `xcom_value` in the return from `poke()` is populated.